coreclient/src/groups/client_auth_info/persistence.rs

```diff
@@ -71,6 +71,62 @@ impl StorableClientCredential {
     }
 }
 
+impl Storable for GroupMembership {
+    // TODO: Reinstate the foreign key constraint as soon as we have migrated
+    // the mls group table to the new style of storage.
+    // FOREIGN KEY (group_id) REFERENCES mlsgroup(primary_key),
```

Comment:
Please make an issue for that and reference it here.

Reply:
Good idea. Done #349